etc/kayobe/inventory/group_vars/compute/network-interfaces

```diff
@@ -19,13 +19,11 @@ internal_interface: "{{ oc_provision_interface }}.{{ internal_vlan }}"
 storage_interface: "{{ oc_provision_interface }}.{{ storage_vlan }}"
 
```

Comment:
Can also remove os_provision_* from here and controllers and storage network-interfaces files.